provider/datagen/src/bin/verify-zero-copy.rs

```diff
@@ -62,31 +60,6 @@ fn main() -> eyre::Result<()> {
                 .long("verbose")
                 .help("Requests verbose output"),
         )
-        .arg(
-            Arg::with_name("POSTCARD_FILE")
```

Comment:
I couldn't be bothered to rewrite this with `SourceData`, given that it's only ever used with testdata. I actually want to make this an integration test in the datagen-library PR, because I don't think this is useful as a public binary.